languages/rust/oso/tests/test_errors.rs

```diff
@@ -89,6 +89,26 @@ fn test_unify_external_not_supported() -> oso::Result<()> {
     Ok(())
 }
 
+/// Test that failing directive sends a sane error message
+#[test]
+fn test_failing_directive() {
```

Comment:
It would be good to rename this `test_failing_inline_query` since we don't use the term directive right now.

Reply:
changed